napari/layers/labels/_tests/test_labels_mouse_bindings.py

```diff
@@ -207,11 +225,13 @@ def test_fill_nD_plane(Event):
     assert np.unique(layer.data[0, 8:10, 8:10]) == 2
 
     layer.mode = 'fill'
-    layer.position = (0, 0, 0)
+    position = (0, 0, 0)
```

Comment:
it answers my question; it can be a 3 tuple...

Reply:
Yes, even nD for an n-dimensional data source